src/index.js

```diff
@@ -102,6 +103,7 @@ export const RCComponent = ({
             <ChatBody
               height={!fullScreen ? height : '83vh'}
               anonymousMode={anonymousMode}
+              showAvatar={showAvatar}
```

Comment:
use `userStore` for this please, directly use it wherever required

Reply:
ohk sorry i will keep this practice in mind